docs/modules/usage_guide/nav.adoc

```diff
@@ -0,0 +1,6 @@
+* xref:security.adoc[]
```

Comment:
I would like to have an index.adoc page for the usage, and then have all these pages as sub pages in there. Have a look at the getting_started nav.adoc for what I mean.

The index.adoc should give an overview over the usage pages.